libexec/bootstrap-scripts/deffile-driver-localimage.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# 
+# Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+#
+# This software is licensed under a 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
+# your rights to use or distribute this software. 
+#
+# 
+
+## Basic sanity
+if [ -z "$SINGULARITY_libexecdir" ]; then
+    echo "Could not identify the Singularity libexecdir."
+    exit 1
+fi
+
+## Load functions
+if [ -f "$SINGULARITY_libexecdir/singularity/functions" ]; then
+    . "$SINGULARITY_libexecdir/singularity/functions"
+else
+    echo "Error loading functions: $SINGULARITY_libexecdir/singularity/functions"
+    exit 1
+fi
+
+if [ -z "${SINGULARITY_ROOTFS:-}" ]; then
+    message ERROR "Singularity root file system not defined\n"
+    exit 1
+fi
+
+if [ -z "${FROM:-}" ]; then
+    message ERROR "Required Definition tag 'From:' not defined.\n"
+    exit 1
+fi
+
+if [ ! -f "${FROM:-}" ]; then
+    message ERROR "${FROM} does not exist\n"
+    exit 1
+fi
+
+if [ -z "${SINGULARITY_BUILDDEF:-}" ]; then
+    exit
+fi
```

Comment:
I copied this script from `deffile-driver-debootstrap.sh` which has the same check also lacking error message or exit code.  

It looks like it is actually possible to run a `bootstrap` without a definition file.  (See [this line in init.c](https://github.com/singularityware/singularity/blob/development/src/bootstrap-lib/init.c#L75) ). For instance, the following works:
```
singularity create lolcow.img
sudo singularity bootstrap lolcow.img docker://godlovedc/lolcow
```
I don't know if this helps to enable that or if it is just an error.  

I actually don't see how the code would ever get into `deffile-driver-localimage.sh` (or `deffile-driver-debootstrap.sh` for that matter) without a definition file because it has to parse the `Bootstrap` field to figure out which script to use.  So this check *may* be safely deleted.  But should probably check with @gmkurtzer to be sure.